community-operators/spark/spark.package.yaml

```diff
@@ -0,0 +1,5 @@
+#! package-manifest: ./deploy/chart/catalog_resources/rh-operators/etcdoperator.v0.9.2.clusterserviceversion.yaml
+packageName: spark-gcp
```

Comment:
The directory should also be changed to match this packageName:
`community-operators/spark-gcp/*`

Reply:
Good catch. Forgot to change that after I copied it over.

Reply:
I've renamed the directory to the packageName and I've removed the `#! package-manifest:` header comment since I'm not sure what that was supposed to be. Plus it seems optional from looking at the other operator bundles.